--- a/app_theme.dart+++ b/app_theme.dart@@ -25,7 +25,7 @@         textStyle: const TextStyle(fontSize: 16, fontWeight: FontWeight.bold),
       )
     ),
-    cardTheme: const CardThemeData( // Corretto: da CardTheme a CardThemeData
+    cardTheme: CardThemeData( // Rimosso 'const'
       elevation: 2,
       shape: RoundedRectangleBorder(
         borderRadius: BorderRadius.circular(12.0),
@@ -57,7 +57,7 @@         textStyle: const TextStyle(fontSize: 16, fontWeight: FontWeight.bold),
       )
     ),
-     cardTheme: const CardThemeData( // Corretto: da CardTheme a CardThemeData
+     cardTheme: CardThemeData( // Rimosso 'const'
       elevation: 4,
       shape: RoundedRectangleBorder(
         borderRadius: BorderRadius.circular(12.0),
